examples/2016-10-31/usage_plan/template.yaml

```diff
@@ -0,0 +1,96 @@
+Conditions:
+  C1:
+    Fn::Equals:
+      - true
+      - true
+
+Parameters:
+  UsagePlanType:
+    Type: String
+    Default: PER_API
+
+Globals:
+    Api:
+      OpenApiVersion: 3.0.0
+      Auth:
+        ApiKeyRequired: true
+        UsagePlan:
+          CreateUsagePlan: !Ref UsagePlanType
+
+Resources:
+   MyApiOne:
+     Condition: C1
+     Type: AWS::Serverless::Api
+     Properties:
+       StageName: Prod
+
+   MyApiTwo:
+     Type: AWS::Serverless::Api
+     Properties:
+       StageName: Prod
+       Auth:
+         UsagePlan:
+          CreateUsagePlan: SHARED
+
+   MyApiTwoUsagePlan:
+      Type: AWS::ApiGateway::UsagePlan
+      DependsOn:
+        - MyApiTwo
+      Properties:
+        ApiStages:
+          - ApiId: !Ref MyApiTwo
+            Stage: !Ref MyApiTwo.Stage
```

Comment:
This doesn't get linked, is there a reason why we're explicitly creating this?

Reply:
Thanks for catching this. You are right this doesn't get used anywhere, I will remove